public/scripts/main/services/loadService.js

```diff
@@ -117,39 +117,92 @@ coaxsApp.service('loadService', function ($http, targetService, supportService)
     $http.get('/geojson/pois')
     .success(function (data, status) {
       if (status == 200) {
+       
         var circles = [];
         var poiUsers = [];
+        var homeIcons = [];
+
+        // var iconColor = ('00000'+(Math.random()*(1<<24)|0).toString(16)).slice(-6)
 
         var smallMarkerOptions = {
-          radius      : 5,
-          fillColor   : 'rgba(139,139,210,0.3)',
-          color       : 'rgba(36,36,76,0.8)',
+          radius      : 10,
+          fillColor   : 'yellow',
+          color       : 'null',
           weight      : 1,
           opacity     : 1,
           fillOpacity : 0.8
         };
+        
+        // Invisible Marker underlay for tooltip activation
         var bigMarkerOptions = {
           radius      : 15,
           fillColor   : 'rgba(139,139,210,0)',
           weight      : 0,
         };
 
+        
+        // POI Marker Class Setup
+        var iconStyle = L.Icon.extend({
+          options : {
+            iconSize     : [25, 25],
+            iconAnchor   : [8, 18],
+            popupAnchor  : [0, -15],
+            opacity: 1,
+            className    : 'icon-on',
+            userId       : 'null'
```

Comment:
more to the pt, why is it a component of the html? why does it take up a whole column? is there a design choice there or should this be rethought (this being the js or the html)?